soc/projects/numerics.md

```diff
@@ -66,24 +68,6 @@ This project proposal is for implementing native Julia algorithms for massively
 
 **Mentors:** [Sacha Verweij](https://github.com/Sacha0)
 
-## Generic linear algebra
-
-Julia supports many different numeric types, both in Base (e.g. `Float32`, `Float64`, `BigFloat`, `Complex`, `Rational`), as well as other packages (e.g. [Quaternions.jl](https://github.com/JuliaGeometry/Quaternions.jl), [ArbFloats.jl](https://github.com/JuliaArbTypes/ArbFloats.jl), [FixedPointNumbers.jl](https://github.com/JuliaMath/FixedPointNumbers.jl), [Unitful.jl](https://github.com/ajkeller34/Unitful.jl)).
```

Comment:
is this intentional?

Reply:
Thanks for pointing that out. I have repaired the damage in the new commit.